Hủaıbaı/assets/minecraft/lang/toa_tqg.json

```diff
@@ -67,6 +67,19 @@
   "selectServer.deleteButton": "Shỉy",
   "selectServer.refresh": "Nỉqguo sủe",
   "selectServer.hiddenAddress": "(Shủı)",
+  "selectWorld.gameMode": "Lủaqcha",
+  "selectWorld.gameMode.survival": "Pó mîefa",
+  "selectWorld.gameMode.creative": "Pó bâı",
+  "selectWorld.gameMode.spectator": "Pó kâqsi",
+  "selectWorld.gameMode.spectator.line1": "Dẻq kảqsı súq da. Keo bủ tỏi ba.",
+  "selectWorld.gameMode.spectator.line2": "",
+  "selectWorld.gameMode.hardcore": "Pó sôq hỉaodaı!",
```

Comment:
I think this would be clearer as just "Sóq hỉaodaı", as in, "Game mode: The most dangerous one"